torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -161,6 +169,10 @@ def tuned_mm_plus_mm(mat1, mat2, mat3, mat4, *, layout=None):
                     **mm_options(config, m1, n1, k1, layout1),
                 )
 
+    if inductor_config.triton.use_gemm_config_lookup_table:
+        if len(choices) > 1:
+            choices = [choices[-1]]
```

Comment:
Can we move some of this to common infra so we aren't needing to duplicate this pattern in every file?